en/config/actuators.md

```diff
@@ -312,25 +312,26 @@ The sliders can power the motors/servos across their full range of motion, and "
 
 :::note
 After you toggle the **Enable sliders** switch, actuators/motors won't do anything until the corresponding slider is _moved_.
-This is a safety feature.
+This is a safety feature to prevent sudden motor movements after switch is enabled.
 :::
 
 Sliders can be used to verify the following:
 
-1. Motors and actuators are assigned to the expected output.
+1. Actuators (Motors, Control surfaces, etc.) are assigned to the expected output.
 1. Motors don't spin when at the `disarmed` PWM output value
 1. Motors barely spin at the `minimum` PWM output value.
 1. Motors give **positive thrust** in the expected direction
-1. Control Surfaces are in the correct idle position when disarmed.
+1. Control Surfaces are in the correct idle position for `disarmed` output value
```

Comment:
Yes, good consistency change. This is what I was trying to capture in my changes - we're slotting to the point that feeds the specified field value :-).